### PR TITLE
[17.0][MIG] mass_mailing_partner: Migration to 17.0

### PR DESCRIPTION
**Pull request by MohamedOsman7**

### CODE REVIEW

**Review by hbrunn — APPROVED** [review]
/ocabot migration mass_mailing_partner

File: mass_mailing_partner/models/mailing_contact_subscription.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# Copyright 2020 Tecnativa - Manuel Calero
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class MailingContactSubscription(models.Model):
```
Comment (CRogos) [inline mass_mailing_partner/models/mailing_contact_subscription.py]: Remove "Contact" from class name and also from file name.

**Review by CRogos — COMMENTED** [review]
Please cleanup commit history.
Code review LGTM.

**Review by CRogos — APPROVED** [review]
Code + Testing LGTM

**Comment by CRogos** [comment]
@pedrobaeza merge is hanging. Could you retry?

**Pull request merged**